modules/virt-about-reviewing-top-consumers.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-dashboard.adoc
+//
+
+[id="virt-about-reviewing-top-consumers_{context}"]
+= About reviewing top consumers of resources
+
+In the {VirtProductName} dashboard, you can select a specific time period and view the top consumers of resources within that time period. Top consumers are virtual machines that are consuming the highest amount of resources.
+
+The following table shows resources monitored in the dashboard and describes the metrics associated with each resource for top consumers.
+
+[cols="1,1"]
+|===
+|*Monitored resources* | *Description*
+|Memory swap traffic
+|Virtual machines consuming the most memory pressure when swapping memory.
+|vCPU wait
+|Virtual machines experiencing the maximum wait time (in seconds) for their vCPUs.
+|CPU usage by pod
+|Pod with virtual machines that are using the maximum CPU usage.
```

Comment:
@lmandavi Virt-launcher pods, that each one is running a Virtual Machine, that are using the most CPU .